stemcell_builder/stages/bosh_ntp/assets/restart.conf

```diff
@@ -4,4 +4,4 @@ PIDFile=/var/run/chronyd.pid
 RemainAfterExit=no
 Restart=always
 RestartSec=5s
-
+InaccessiblePaths=/var/vcap/store
```

Comment:
Should we add a `-` prefix in case when the persistent disk does not exist?

See https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ReadWritePaths=
 
_"Paths in ReadWritePaths=, ReadOnlyPaths=, InaccessiblePaths=, ExecPaths= and NoExecPaths= may be prefixed with "-", in which case they will be ignored when they do not exist."_

Reply:
i just tested all scenarios.
and when i add a `-` /var/vcap/store is being added again.

without the prefix both mounted and umounted works as designed

Reply:
@ramonskie does chrony also start on instances without a persistent disk?

Reply:
@rkoster yes i tested the following.
- with persistent disk +prefix - var/vcap/store shows up in the ns again
- without persistent disk +prefix

- with persistent disk +without the prefix (current) - var/vcap/store does not shows up in the ns again
- without persistent disk +without the prefix (current)

Reply:
a second quicktest. i noticed a flaw in my test.
and the prefix is needed.
i will fix that in another pr